ci/do.sh

```diff
@@ -150,21 +141,41 @@ fi
 
 if [ "x${FIRST_TIME_BUILD}" == "x1" -a "x${IS_CI_BUILD}" == "x1" ]; then
   echo "### First time build is complete ###"
+  host=$(hostname -s)
+  echo "READY:${host}" >>${config_dir}/.status
   exit 0
 fi
 
+if [ "x${ID}" == "xcentos" -a "x${VERSION_ID}" == "x8" ]; then
+  if [ ! -f /tmp/swig/swig/configure ]; then
+    # source install swig
+    dnf -y install gcc-c++ byacc pcre-devel
+    mkdir -p /tmp/swig/
+    cd /tmp/swig
+    git clone https://github.com/swig/swig --branch rel-4.0.0 --single-branch
+    cd swig
+    ./autogen.sh
+    ./configure
```

Comment:
I think using CFLAGS will help.

Reply:
This is configure for 'swig' following the same steps as prescribed for a successful installation.